var/spack/repos/builtin/packages/ramses/package.py

```diff
@@ -0,0 +1,54 @@
+# Copyright 2013-2022 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+import os
+
+from spack.package import *
+
+
+class Ramses(MakefilePackage):
+    """Ramses benchmark for DiRAC.
+
+    The source code for this benchmark is stored in a private repository. To
+    gain access please contact the RSE team at the University of Leicester or
+    contact via github from our organization page
+    https://github.com/UniOfLeicester
+    """
+
+    homepage = "https://github.com/UniOfLeicester/benchmark-ramses"
+    git = "ssh://git@github.com/UniOfLeicester/benchmark-ramses.git"
```

Comment:
This answers both this question and the one above (re: `404 Page not found`).

These repos are private because the source code is not opensource. We use them for benchmarking our HPC facilities. Are we allowed to add packages to spack that have private repos? 

As a result we need to use `ssh://` instead of `https://`.

Reply:
> Are we allowed to add packages to spack that have private repos?

Great question.  Let me ask @tgamblin